code/game/machinery/bots/ed209bot.dm

```diff
@@ -247,7 +247,7 @@ Auto Patrol: []"},
 	..()
 
 	threatlevel = H.assess_threat(src)
-	threatlevel += 6
+	threatlevel += (PERP_LEVEL_ARREST + PERP_LEVEL_ARREST/2)
```

Comment:
For stuff like this you should probably make a second define, not reuse the old one.